open_samus_returns_rando/__init__.py

```diff
@@ -1 +0,0 @@
-from .samus_returns_patcher import patch_extracted
```

Comment:
I still don't know about thus @henriquegemignani

Reply:
Dread does this which is obviously more fleshed out https://github.com/randovania/open-dread-rando/blob/main/src/open_dread_rando/__init__.py